domain/test/com/thoughtworks/go/domain/materials/MaterialsTest.java

```diff
@@ -74,7 +75,7 @@ public void shouldGetMaterialByFolder() {
         materials.add(material1);
         materials.add(material2);
 
-        assertThat((HgMaterial) materials.byFolder("folder1"), is(material1));
+        assertThat((HgMaterial) materials.byFolder("folder1"), Is.is(material1));
```

Comment:
Did you change some setting in IntelliJ to disable static imports? Did you mean to do this? I see it done a lot. :)